administrator/components/com_content/tmpl/articles/default.php

```diff
@@ -154,6 +156,9 @@
                               endif; ?>>
                         <?php foreach ($this->items as $i => $item) :
                             $item->max_ordering = 0;
+                            $ordering             = ($listOrder == 'fp.ordering');
+							$assetId              = 'com_content.article.' . $item->id;
+							$canCreate            = $user->authorise('core.create',     'com_content.category.' . $item->catid);
```

Comment:
```suggestion
                            $ordering             = ($listOrder == 'fp.ordering');
                            $assetId              = 'com_content.article.' . $item->id;
                            $canCreate            = $user->authorise('core.create',     'com_content.category.' . $item->catid);
```
Spaces and not tabs.